app/views/tracks.js

```diff
@@ -1,38 +1,33 @@
 import Mn from 'backbone.marionette';
-import Tracks from '../collections/tracks';
 import TrackView from './track';
+import application from '../application';
 
-const TracksView = Mn.CollectionView.extend({  
-    el: '#app-hook',
-    tagName: 'ul',
+
+const TracksView = Mn.CompositeView.extend({
+
+    template: require('./templates/tracks'),
+
+    childViewContainer: '#track-list',
 
     childView: TrackView,
-    events: {
-        'click a': 'play',
-        'click .delete': 'delete'
-    },
-    play: function (e) {
-        const id = e.currentTarget.dataset.id;
-        const item = this.collection.get(id);
-        item.getStreamURL(playAudio);
-    },
-    delete: function (e) {
-        const id = e.currentTarget.dataset.id;
-        const item = this.collection.get(id);
-        item.set('hidden', true);
-        item.save();
+
+    onRender: function() {
+        this.setCurrentTrack();
+        this.listenTo(
+            application.upNext,
+            'change:currentTrack',
+            this.setCurrentTrack
+        );
     },
-    initialize: function() {
-        const tracks = new Tracks();
-        tracks.fetch();
-        this.collection = tracks;
+
+    setCurrentTrack: function() {
+        if (application.upNext && application.upNext.getAttr('currentTrack')) {
+            const track = application.upNext.getAttr('currentTrack');
```

Comment:
`upNext` collection should not have attributes. Use `appState` to store application state